src/levels.rs

```diff
@@ -81,10 +81,12 @@ pub const DCT_ADST: TxfmType = 2;
 pub const ADST_DCT: TxfmType = 1;
 pub const DCT_DCT: TxfmType = 0;
 
-pub type TxClass = c_uint;
-pub const TX_CLASS_V: TxClass = 2;
-pub const TX_CLASS_H: TxClass = 1;
-pub const TX_CLASS_2D: TxClass = 0;
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub enum TxClass {
+    TwoD = 0,
+    H = 1,
+    V = 2,
```

Comment:
Do we need the specified values since we're not parsing this from numbers (e.x. `from_repr`)?

Reply:
we don't, fixed